build-farm/platform-specific-configurations/mac.sh

```diff
@@ -53,11 +53,28 @@ then
   export JDK_BOOT_DIR=$JDK10_BOOT_DIR
 fi
 
+if [ "${JAVA_TO_BUILD}" == "${JDK12_VERSION}" ] || [ "${JAVA_TO_BUILD}" == "${JDKHEAD_VERSION}" ]
+then
+  if [ "${VARIANT}" == "${BUILD_VARIANT_OPENJ9}" ]; then
+    export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-openssl=fetched --enable-openssl-bundling"
+  else
+    export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-extra-cxxflags=-mmacosx-version-min=10.8"
+  fi
+  if [ ! -d "$JDK11_BOOT_DIR" ]; then
+    export JDK11_BOOT_DIR="$PWD/jdk-11"
+    if [ ! -d "$JDK11_BOOT_DIR/bin" ]; then
+      mkdir -p "$JDK11_BOOT_DIR"
+      wget -q -O - 'https://api.adoptopenjdk.net/v2/binary/releases/openjdk11?os=mac&release=latest&type=jdk&heap_size=normal&openjdk_impl=openj9' | tar xpzf - --strip-components=2 -C "$JDK11_BOOT_DIR"
```

Comment:
again this should probably be hotspot

Reply:
I don't mind as long as an appropriate unique build is returned.  BTW, the preexisting jdk11 code doesn't work because it returns multiple builds, so the builds must have been relying on the boot jdk already present on the build machines.

Reply:
Changed to hotspot